common/package.json

```diff
@@ -0,0 +1,9 @@
+{
+	"name": "@daydream/common",
+	"version": "1.0.0",
+	"main": "dist/index",
+	"dependencies": {
+		"next": "^13.0.2",
+		"typescript": "^4.8.4"
```

Comment:
Some things here.
- This package doesnt use next and as such it shouldnt be specified here
- We are using next 12. Next 13 is technically the same but some imports would need to be fixed
- typescript should be a dev dependency